src/pytest_param_scope/plugin.py

```diff
@@ -44,7 +47,17 @@ def param_scope(request):
 
```

Comment:
Hmm, I'm not sure how to comment on un-diffed areas in a PR review! But anyways, a few lines above this where the `if m:` clause appears, the case of excess teardown could be handled earlier there, rather than handling it on the line comment below.